src/components/Hoverable/types.ts

```diff
@@ -13,12 +13,6 @@ type HoverableProps = {
     /** Function that executes when the mouse leaves the children. */
     onHoverOut?: () => void;
 
-    /** Direct pass-through of React's onMouseEnter event. */
-    onMouseEnter?: (event: MouseEvent) => void;
-
-    /** Direct pass-through of React's onMouseLeave event. */
-    onMouseLeave?: (event: MouseEvent) => void;
-
```

Comment:
Callbacks removed ✅

Reply:
That's correct 👍

Reply:
This comment is in the context of [the "checklist"](https://github.com/Expensify/App/issues/32096#issuecomment-1829651109) prepared by Vit. It would be good to ensure all of its parts are handled as required.

Reply:
Ah right, that's actually a checklist written by me in order to track all potential changes :D 

I anwered your questions there 👍

Reply:
Oh, fine. I thought this was from Expensify's side. Thank you